disk/block_test.go

```diff
@@ -0,0 +1,151 @@
+package disk
+
+import (
+	"bytes"
+	"errors"
+	"io"
+	"os"
+	"strings"
+	"testing"
+)
+
+const (
+	tmpTestFile = "/tmp/cfs_disk_block_test"
```

Comment:
`TempFile` calls
`os.OpenFile(name, os.O_RDWR|os.O_CREATE|os.O_EXCL, 0600)`
But we do want to use `os.O_TRUNC` to reset the file every time we set up tests.